plugins/catalog-backend-module-bitbucket-cloud/src/providers/BitbucketCloudEntityProvider.ts

```diff
@@ -234,6 +239,8 @@ export class BitbucketCloudEntityProvider
       throw new Error('Not initialized');
     }
 
+    this.enhanceEvent(event);
```

Comment:
It's a bit indirect isn't it, to compensate for a field that we know will be missing, by generating it in the original model - instead of the consuming code reading the field that we we know will be present 🤔 But I assume there's a reason

Reply:
that's how I started actually. The APIs return this field and some of the filtering logic depends on it. It just felt like a minimal fix.